kevinearldenny/geodjango-demo-toy
# Derek Review Feedback

## Overview

I went through and either made changes or added comments for places where I had feedback.

## Notes

In general, there wasn't much that was "wrong" given that this is a toy project, but I added comments to highlight the types of things we would be doing if this was a client project. I'm going to go through and add some comments to my changes after opening this, as well.

## Testing instructions

- Run `docker-compose build` and confirm everything works
- Run `docker-compose up` and confirm the map page appears
- Load data according to the README instructions, and confirm neighborhoods appear on the map
